editor/src/components/navigator/navigator-utils.ts

```diff
@@ -75,7 +75,7 @@ export function navigatorDepth(
     isFeatureEnabled('Conditional support') &&
     isConditionalClauseNavigatorEntry(navigatorEntry)
   ) {
-    result = result - 1
+    result = result + 1
```

Comment:
???

Reply:
Ah, because there's a tweak I made related to the conditional clause entry not having the path down into the clause anymore:
https://github.com/concrete-utopia/utopia/pull/3426/files#diff-9927081e43e2bc863911820ed9808e422cbd44f73d7ef5229db0b2999d073d77R152